packages/app-bridge/src/tests/ColorDummy.ts

```diff
@@ -6,108 +6,127 @@ export class ColorDummy {
     static red(id = 767): Color {
         return {
             id,
-            name: 'Red',
             sort: 1,
-            nameCss: 'red',
-            hex: 'ff0000',
-            red: 255,
-            green: 0,
-            blue: 0,
-            alpha: 255,
-            hue: 361,
-            saturation: 100,
-            lightness: 46,
-            c: 0,
-            m: 100,
-            y: 100,
-            k: 0,
-            pantone: null,
-            ral: null,
-            oracal: null,
-            pantoneCoated: null,
-            pantoneUncoated: null,
-            cmykCoated: null,
-            cmykUncoated: null,
-            cmykNewspaper: null,
-            ncs: null,
-            pantoneCp: null,
-            pantonePlastics: null,
-            pantoneTextile: null,
-            hks: null,
-            threeM: null,
-            lab: null,
+            title: 'Red',
+            revision: {
+                nameCss: 'red',
+                hex: 'ff0000',
+                hue: 361,
+                saturation: 100,
+                lightness: 46,
+                rgba: { red: 255, green: 0, blue: 0, alpha: 255 },
+                cmyk: {
+                    cyan: 0,
+                    magenta: 100,
+                    yellow: 100,
+                    key: 0,
```

Comment:
K is for blacK, not key ... I know it's weird but I didn't come up with the idea 😅 
Please just rename key to black